ui/src/app/workflows/components/workflows-row/workflows-row.tsx

```diff
@@ -41,16 +42,16 @@ export function WorkflowsRow(props: WorkflowsRowProps) {
                     />
                     <PhaseIcon value={wf.status.phase} />
                 </div>
+                <div className='columns small-2'>
+                    <a className='overlay' href={workflowLink}></a>
```

Comment:
this is indeed quite hacky as you mentioned in the PR description. I also can't seem to expand the row right now, clicking the "Show" button takes me to the Workflow Details page.

instead of using an `a` tag, you can use a different element and capture the event. clicks elsewhere should capture the event first and prevent it from bubbling up to this level.

but tbh, I don't like that the whole thing is a link in the first place, that's very confusing UX-wise as there are different behaviors based on where/when you click. so I actually think we should change up the behavior anyway, but I'm not yet quite sure where to put the link to the details page then

Reply:
So I removed the whole row being a link in #12580 and made the name only a link.
But then the nested linking issue re-appeared, so I used JS as I mentioned above to prevent bubble up, which is a lot less hacky and less problematic then the overlay CSS (although it still has some semantic issues, as I mention in #12580, but they're technically an edge of an edge case)